packages/connection-form/src/components/connect-form.tsx

```diff
@@ -128,104 +132,117 @@ function ConnectForm({
     (error) => error.fieldName === 'connectionString'
   );
 
+  const onSubmitForm = useCallback(() => {
+    const updatedConnectionOptions = cloneDeep(connectionOptions);
+    const formErrors = validateConnectionOptionsErrors(
+      updatedConnectionOptions
+    );
+    if (formErrors.length) {
+      setErrors(formErrors);
+      return;
+    }
```

Comment:
I’m sorry I didn’t see this before, can we maybe merge this a bit with the code that’s now in https://github.com/mongodb-js/compass/blob/4daa8ef4bd75abce4748c4844eebb04748328d93/packages/connection-form/src/components/connect-form.tsx#L135-L142?

Reply:
Merged in main 👍